services/networking/loadbalancer/features.yaml

```diff
@@ -1,7 +1,3 @@
-title: CCC Load Balancer Features
-category_id: CCC.LB
-type: Networking
-description: This documents the minimual set of features that should be present for a load balancer service to be considered for use in financial services ecosystems.
 features:
```

Comment:
we need `common_features` section before `features`

Reply:
```suggestion
common_features: []
features:
```